tests/python-pytest/onnx/export/mxnet_export_test.py

```diff
@@ -238,6 +251,70 @@ def test_square():
 
     npt.assert_almost_equal(result, numpy_op)
 
+@with_seed()
+def test_logical_and():
```

Comment:
for these simple operators, instead of repeating test code for each of them, can you create a map[operator] = (input, output) and run all operators tests in one test method?